src/neuroconv/datainterfaces/ecephys/edf/requirements.txt

```diff
@@ -1,2 +1 @@
-pyedflib>=0.1.30
-numpy<1.25.0;python_version>="3.11"
```

Comment:
So looks like `pyedlib` fixed the source issue themselves at some point after our safety pin

Reply:
Great!